Minimal example fails to compile under Linux

The following compilation errors are shown:
```
src/generated.cpp:823:18: error: ‘class wxWindow’ has no member named ‘RegisterHotKey’
src/generated.cpp:826:18: error: ‘class wxWindow’ has no member named ‘UnregisterHotKey’
```

I think this happens because those methods are only available on Windows.

--- a/doxybindgen.toml
+++ b/doxybindgen.toml
@@ -38,6 +38,10 @@ blocklist = [
     'GetChildren1',
     'GetCursor',
     'GetUpdateRegion',
+
+    # wx3.1 on wxGTK
+    'RegisterHotKey',
+    'UnregisterHotKey',
 ]
 blocklist30 = [
     'DisableFocusFromKeyboard',
@@ -52,14 +56,12 @@ blocklist30 = [
     'GetDPI',
     'GetDPIScaleFactor',
     'InheritsForegroundColour',
-    'RegisterHotKey',
     'ToDIP',
     'ToDIP1',
     'ToDIP2',
     'ToDIP3',
     'ToDIP4',
     'ToDIP5',
-    'UnregisterHotKey',
     'UseBackgroundColour',
     'UseForegroundColour',
 ]

--- a/wx-core/include/generated.h
+++ b/wx-core/include/generated.h
@@ -289,10 +289,6 @@ bool wxWindow_IsThisEnabled(const wxWindow * self);
 bool wxWindow_IsTopLevel(const wxWindow * self);
 void wxWindow_OnInternalIdle(wxWindow * self);
 bool wxWindow_SendIdleEvents(wxWindow * self, wxIdleEvent * event);
-#if wxCHECK_VERSION(3, 1, 0)
-bool wxWindow_RegisterHotKey(wxWindow * self, int hotkey_id, int modifiers, int virtual_key_code);
-bool wxWindow_UnregisterHotKey(wxWindow * self, int hotkey_id);
-#endif
 void wxWindow_UpdateWindowUI(wxWindow * self, long flags);
 wxWindow * wxWindow_FindFocus();
 wxWindow * wxWindow_FindWindowById(long id, const wxWindow * parent);

--- a/wx-core/src/generated.cpp
+++ b/wx-core/src/generated.cpp
@@ -818,14 +818,6 @@ void wxWindow_OnInternalIdle(wxWindow * self) {
 bool wxWindow_SendIdleEvents(wxWindow * self, wxIdleEvent * event) {
     return self->SendIdleEvents(*event);
 }
-#if wxCHECK_VERSION(3, 1, 0)
-bool wxWindow_RegisterHotKey(wxWindow * self, int hotkey_id, int modifiers, int virtual_key_code) {
-    return self->RegisterHotKey(hotkey_id, modifiers, virtual_key_code);
-}
-bool wxWindow_UnregisterHotKey(wxWindow * self, int hotkey_id) {
-    return self->UnregisterHotKey(hotkey_id);
-}
-#endif
 void wxWindow_UpdateWindowUI(wxWindow * self, long flags) {
     return self->UpdateWindowUI(flags);
 }

--- a/wx-core/src/generated.rs
+++ b/wx-core/src/generated.rs
@@ -309,8 +309,8 @@ mod ffi {
         pub fn wxWindow_IsTopLevel(self_: *const c_void) -> bool;
         pub fn wxWindow_OnInternalIdle(self_: *mut c_void);
         pub fn wxWindow_SendIdleEvents(self_: *mut c_void, event: *mut c_void) -> bool;
-        pub fn wxWindow_RegisterHotKey(self_: *mut c_void, hotkey_id: c_int, modifiers: c_int, virtual_key_code: c_int) -> bool;
-        pub fn wxWindow_UnregisterHotKey(self_: *mut c_void, hotkey_id: c_int) -> bool;
+        // BLOCKED: pub fn wxWindow_RegisterHotKey(self_: *mut c_void, hotkey_id: c_int, modifiers: c_int, virtual_key_code: c_int) -> bool;
+        // BLOCKED: pub fn wxWindow_UnregisterHotKey(self_: *mut c_void, hotkey_id: c_int) -> bool;
         pub fn wxWindow_UpdateWindowUI(self_: *mut c_void, flags: c_long);
         // NOT_SUPPORTED: pub fn wxWindow_GetClassDefaultAttributes(variant: wxWindowVariant) -> wxVisualAttributes;
         pub fn wxWindow_FindFocus() -> *mut c_void;
@@ -1747,12 +1747,8 @@ pub mod methods {
         fn send_idle_events(&self, event: *mut c_void) -> bool {
             unsafe { ffi::wxWindow_SendIdleEvents(self.as_ptr(), event) }
         }
-        fn register_hot_key(&self, hotkey_id: c_int, modifiers: c_int, virtual_key_code: c_int) -> bool {
-            unsafe { ffi::wxWindow_RegisterHotKey(self.as_ptr(), hotkey_id, modifiers, virtual_key_code) }
-        }
-        fn unregister_hot_key(&self, hotkey_id: c_int) -> bool {
-            unsafe { ffi::wxWindow_UnregisterHotKey(self.as_ptr(), hotkey_id) }
-        }
+        // BLOCKED: fn RegisterHotKey()
+        // BLOCKED: fn UnregisterHotKey()
         fn update_window_ui(&self, flags: c_long) {
             unsafe { ffi::wxWindow_UpdateWindowUI(self.as_ptr(), flags) }
         }

